morerss/zhihu_stream.py

```diff
@@ -69,6 +100,30 @@ class ZhihuAPI:
       'url': urljoin('https://www.zhihu.com/', url),
     }
 
+  async def topic_intro(self, name):
+    """
+    Zhihu topic information
+    :param name: Zhihu topic slug
+    :return:
+    """
+    intro_url = 'https://www.zhihu.com/api/v4/topics/%s/intro' % name
+    res = await fetch_zhihu(
+      intro_url, headers={'User-Agent': self.user_agent})
+    if not res.body:
+      # e.g. https://www.zhihu.com/bei-feng-san-dai
+      raise web.HTTPError(404)
+    intro = json.loads(res.body.decode('utf-8'))
+    # topic name
+    name = intro['content']['meta']['name']
```

Comment:
I got `KeyError: 'meta'` on this line, e.g. https://www.zhihu.com/api/v4/topics/19667817/intro doesn't have a meta key.

Reply:
Zhihu's API has some weird inconsistencies. :frowning_face:  I'll find a workaround.

Reply:
https://github.com/lilydjwg/morerssplz/pull/32/commits/6d01a6e1a23ba5f68628e03cc52a522b276f67c8 fixes the missing topic name.